xwalk_core_library_android.gypi

```diff
@@ -441,7 +441,13 @@
           ],
           'action': [
             'python', '<(DEPTH)/xwalk/build/android/generate_xwalk_core_library_aar.py',
-            '-t', '<(PRODUCT_DIR)',
+            '--aar-path', '<(PRODUCT_DIR)/xwalk_core_library.aar',
+            '--android-manifest', '<(DEPTH)/xwalk/build/android/xwalkcore_library_template/AndroidManifest.xml',
+            '--classes-jar', '<(PRODUCT_DIR)/lib.java/xwalk_core_library_java.jar',
+            '--jni-abi', '<(android_app_abi)',
+            '--jni-dir', '<(PRODUCT_DIR)/xwalk_core_library/libs/<(android_app_abi)',
+            '--res-dir', '<(PRODUCT_DIR)/xwalk_core_library/res',
+            '--r-txt', '<(PRODUCT_DIR)/xwalk_core_internal_empty_embedder_apk/gen/R.txt',
```

Comment:
In original logic it is "xwalk_core_empty_embedder_apk/gen/R.txt" but here it is "xwalk_core_**internal**_empty_embedder_apk"?